ex9/js/main.js

```diff
@@ -0,0 +1,95 @@
+require(['ajaxGet'], function(ajaxGet) {
+  $(function() {
```

Comment:
I would actually inline (=move the code of something into here) setEvents into the `$(function() {...})`, but move the `on` outside. Also, I don't think the `on` object helps in any way, I'd vote to just its methods be top-level functions.